python/mim_control/impedance_controller.py

```diff
@@ -15,7 +15,13 @@
 
 class ImpedanceController(object):
     def __init__(
-        self, name, pin_robot, frame_root_name, frame_end_name, start_column
+        self,
+        name,
+        pin_robot,
+        frame_root_name,
+        frame_end_name,
+        start_column,
+        is_joint_active,
```

Comment:
I don't understand what `is_joint_active` is taking as values. From the way it's used below it seems to be a list. However, if a variable starts with `is`, I would assume it's a boolean value instead of a list.

Reply:
It's an array of boolean.

Reply:
How about calling it then `active_joints` instead?

Also, if you want to add this new property, please update all the robot config files that are using it. Or add a default parameter which makes it act like the current code by default if no `active_joints` is passed in.

Reply:
I will change it to active_joints and create pull requests for all of the robots.